### PR TITLE
publishes odometry messages based on localization data

### PR DESCRIPTION
**Pull request by jrgnicho**
This commits computes odometry data based on the measured sensor location and publishes an nav/Odometry message

### CODE REVIEW

File: ros/sicknav350/sicknav350.cpp [hunk 1]
```diff
@@ -108,6 +123,30 @@ class smoothtime {
   }
 };
 
+void createOdometryMessage(const ros::Duration& time_elapsed, const tf::Transform& prev_transform,
```
Comment (shaun-edwards) [inline ros/sicknav350/sicknav350.cpp:126]: Doesn't `odom` have a covariance matrix?  Does this need to be populated? (just curious)

**Comment by jrgnicho** [comment]
The simulation had all values of this array set to zero.  It'd probably be safe to initialize them to zero so that they don't get initialized to other values by default.

**Comment by jrgnicho** [comment]
@shaun-edwards please review

**Comment by shaun-edwards** [comment]
+1

**Pull request merged**